packages/core/src/components/context-menu/contextMenu.tsx

```diff
@@ -56,6 +52,7 @@ class ContextMenu extends AbstractPureComponent<IContextMenuProps, IContextMenuS
         return (
             <div className={Classes.CONTEXT_MENU_POPOVER_TARGET} style={this.state.offset}>
                 <Popover
+                    {...this.props}
```

Comment:
this seems like a pretty major change where users could possible pass props to the popover this way, now they can't.

Reply:
this component is not even exported, so it think we're safe here. users use either a decorator `@ContextMenuTarget` or a static method `ContextMenu.show()` (which looks like this component but isn't).